meson.build

```diff
@@ -80,21 +80,12 @@ common_cflags = cc.get_supported_arguments(test_cflags)
 
 message('Building DCV Color Primitives')
 
-cargo_env = {}
-
-if get_option('buildtype') == 'release'
-  dcp_build_dir = 'release'
-  cargo_target = ['build', '--release', ]
-elif get_option('buildtype') == 'debugoptimized'
-  dcp_build_dir = 'release'
-  cargo_target = ['build', '--release', ]
-  cargo_env = { 'RUSTFLAGS': '-g' }
-else
-  dcp_build_dir = 'debug'
-  cargo_target = ['build', ]
-endif
+dcp_build_dir = get_option('buildtype')
+
+cargo_buildtype = '--@0@'.format(get_option('buildtype'))
+cargo_target = ['build', cargo_buildtype, ]
```

Comment:
If `buildtype` is set to `debugoptimized`, we executed:
`cargo build --debugoptimized`

In this case, we want to execute:
`cargo build release`

---

If `buildtype` is set to `debug`, we executed:
`cargo build --debug`

In this case, we want to execute:
`cargo build`

Can we check this?

Reply:
PR updated